llm/llama-3_2/README.md

```diff
@@ -0,0 +1,356 @@
+
+# Vision Llama-3.2 (Meta) 
+
+
+<p align="center">
+<img src="https://i.imgur.com/1NEZs9f.png" alt="Llama-3 x SkyPilot" style="width: 50%;">
+</p>
+
+[Llama 3.2](https://ai.meta.com/blog/llama-3-2-connect-2024-vision-edge-mobile-devices/) is out on the date of Sep 25th 2024. It not only includes the latest models for chat, but also includes the multimodal models. Let's point and launch it with SkyPilot.
+
+* [Llama 3.2 release](https://ai.meta.com/blog/llama-3-2-connect-2024-vision-edge-mobile-devices/)
+
+
+
+## Why use SkyPilot?
+
+* Point, launch, and serve: simply point to the cloud/kubernetes cluster you have access to, and launch the model there with a single command.
+* No lock-in: run on any supported cloud - AWS, Azure, GCP, Lambda Cloud, IBM, Samsung, OCI
+* Everything stays in your cloud account (your VMs & buckets)
+* No one else sees your chat history
+* Pay absolute minimum — no managed solution markups
+* Freely choose your own model size, GPU type, number of GPUs, etc, based on scale and budget.
+
+…and you get all of this with 1 click — let SkyPilot automate the infra.
+
+
+## Prerequisites
+
+- Go to the [HuggingFace model page](https://huggingface.co/meta-llama/) and request access to the model [meta-llama/Llama-3.2-1B-Instruct](https://huggingface.co/meta-llama/Llama-3.2-1B-Instruct) and [meta-llama/Llama-3.2-11B-Vision](https://huggingface.co/meta-llama/Llama-3.2-11B-Vision).
+- Check that you have installed SkyPilot ([docs](https://skypilot.readthedocs.io/en/latest/getting-started/installation.html)).
+- Check that `sky check` shows clouds or Kubernetes are enabled.
+
+## SkyPilot YAML
+
```

Comment:
Maybe add a quick table/bullet list of all Llama 3.2 variants, and hint users to replace `envs.MODEL_NAME`.